ocaml/sdk-gen/csharp/templates/XenServer.csproj.mustache

```diff
@@ -1,96 +1,32 @@
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
-    <DebugType>pdbonly</DebugType>
-    <Optimize>true</Optimize>
-    <OutputPath>bin\Release\</OutputPath>
-    <DefineConstants>TRACE</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
-    <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="CookComputing.XmlRpcV2">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>.\CookComputing.XmlRpcV2.dll</HintPath>
-    </Reference>
     <Reference Include="Newtonsoft.Json.CH.dll">
       <SpecificVersion>False</SpecificVersion>
       <HintPath>.\Newtonsoft.Json.CH.dll</HintPath>
     </Reference>
-    <Reference Include="System" />
-    <Reference Include="System.Data" />
-    <Reference Include="System.Xml" />
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="Properties\AssemblyInfo.cs" />
-{{#api_members}}
-    <Compile Include="{{api_member}}.cs" />
-{{/api_members}}
-    <Compile Include="ApiVersion.cs" />
-    <Compile Include="Event.cs" />
-    <Compile Include="DeprecatedAttribute.cs" />
-    <Compile Include="Failure.cs" />
-    <Compile Include="Helper.cs" />
-    <Compile Include="HTTP.cs" />
-    <Compile Include="IMockWebProxy.cs" />
-    <Compile Include="IXenObject.cs" />
-    <Compile Include="Converters.cs" />
-    <Compile Include="JsonRpc.cs" />
-    <Compile Include="JsonRpcClient.cs" />
-    <Compile Include="Marshalling.cs" />
-    <Compile Include="Message2.cs" />
-    <Compile Include="Response.cs" />
-    <Compile Include="Session.cs" />
-    <Compile Include="Subject2.cs" />
-    <Compile Include="UserDetails.cs" />
-    <Compile Include="XenObject.cs" />
-    <Compile Include="XenRef.cs" />
-    <Compile Include="Relation.cs" />
-    <Compile Include="HTTP_actions.cs" />
-    <Compile Include="Proxy.cs" />
-    <Compile Include="Maps.cs" />
-    <Compile Include="FriendlyErrorNames.Designer.cs">
+    <Compile Update="FriendlyErrorNames.Designer.cs">
       <AutoGen>True</AutoGen>
       <DesignTime>True</DesignTime>
       <DependentUpon>FriendlyErrorNames.resx</DependentUpon>
     </Compile>
   </ItemGroup>
   <ItemGroup>
-    <EmbeddedResource Include="FriendlyErrorNames.resx">
+    <EmbeddedResource Update="FriendlyErrorNames.resx">
       <SubType>Designer</SubType>
       <Generator>ResXFileCodeGenerator</Generator>
       <CustomToolNamespace>XenAPI</CustomToolNamespace>
       <LastGenOutput>FriendlyErrorNames.Designer.cs</LastGenOutput>
     </EmbeddedResource>
   </ItemGroup>
-  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-  <!-- To modify your build process, add your task inside one of the targets below and uncomment it.
-       Other similar extension points exist, see Microsoft.Common.targets.
-  <Target Name="BeforeBuild">
-  </Target>
-  <Target Name="AfterBuild">
-  </Target>
-  -->
-</Project>
+</Project>
```

Comment:
Looks like the generated file has it, too. Must be a copy/paste error. I'll add it to avoid the same confusion appearing in case we need to amend the file in the future 💯